gr-utils/python/modtool/cli/add.py

```diff
@@ -109,7 +110,16 @@ def get_blockname(self):
 def get_copyrightholder(self):
     """ Get the copyrightholder of the block to be added """
     if self.info['copyrightholder'] is None:
-        self.info['copyrightholder'] = '<+YOU OR YOUR COMPANY+>'
+        user = getpass.getuser()
```

Comment:
I don't think this is worth the cost of another dependency.

Reply:
I am not very sure but I think `getpass` is a internal python library.

Reply:
Ah alrighty.